Makefile.am

```diff
@@ -225,6 +225,7 @@ EXTRA_DIST += \
     test/util/data/blanktxv1.json \
     test/util/data/blanktxv2.hex \
     test/util/data/blanktxv2.json \
+    test/util/rpcauth-test.py \
```

Comment:
nit: seems like somewhat random placement. Why not alphabetically sorted like the rest?

Reply:
Done